### ISSUE TITLE
Project Created via Terraform Not Visible in Web UI for Organization Root User

### ISSUE THREAD

**Issue by fsz-codeshop** [Infisical/terraform-provider-infisical#40]
### TLDR;
**Problem Statement:**
After creating a project using Terraform, the project does not appear in the web UI when accessed by the organization root user.

**Reproducible Steps:**

1. Use Terraform to create a new project within the organization.
2. Log in to the web UI using the organization root user credentials.
3. Navigate to the section where projects are listed.

**Expected Outcome:**
The organization root user should be able to see all projects within the organization, including the newly created project.

### Full Description
After creating a project via terraform, the project is not showing up in web UI:
![image](https://github.com/Infisical/terraform-provider-infisical/assets/79223916/6b889337-fb7f-452b-8766-ca7dc1faa111)
![image](https://github.com/Infisical/terraform-provider-infisical/assets/79223916/2646f3f3-febb-484d-90fb-6e57ee33430b)

My user is full-admin:
![image](https://github.com/Infisical/terraform-provider-infisical/assets/79223916/92c291d4-8ad8-45ab-84d7-2fc9e8dddd94)

And according to the roled escription, it should have full-powers over projects:
![image](https://github.com/Infisical/terraform-provider-infisical/assets/79223916/1ebb2330-9617-4783-99db-8595c62fc2d3)

I had to add my full-admin user to the project via terraform to be able to see it via web UI:
![image](https://github.com/Infisical/terraform-provider-infisical/assets/79223916/24814776-be5f-4b7b-80ca-b26d4021de70)
![image](https://github.com/Infisical/terraform-provider-infisical/assets/79223916/831bfc46-a1ae-4f65-af73-2e9c75f9d535)

**Comment by valentindiehl** [comment]
I have the same issue unfortunately, also secrets created don't show up in the web UI and also projects created through terraform can't be selected in the CLI

**Comment by akhilmhdh** [comment]
This is expectec behavoir. The project created via terraform the only member is the tf identity. 

To access it as org admin refer this:
https://infisical.com/docs/documentation/platform/admin-panel/org-admin-console#accessing-a-project-in-your-organization
